dali/image/jpeg.cc

```diff
@@ -15,7 +15,9 @@
 #include "dali/image/jpeg.h"
 #include <cmath>
 #include <memory>
+#if !defined(__AARCH64_GNU__) && !defined(__AARCH64_QNX__)
```

Comment:
I am wondering, what was the issue with `jpeg_mem.h`?

Reply:
It includes jpeg_utils which further includes jerror  and jpeglib headers which I believe belong to turbojpeg library not available for non x86_64 platform

Reply:
libjpeg-turbo is built from source in our Dockerfile.deps, and probably you can do the same in your docker

Reply:
added Dockerfile